get-dependencies_linux.sh

```diff
@@ -76,7 +76,7 @@ if [ "${DISTRO}" == "fedora" ]; then
   fi
 
   echo "Installing build-dependencies for Fedora"
-  dnf -y install cmake git p7zip gettext rubygem-asciidoctor SDL2-devel physfs-devel libpng-devel openal-soft-devel libvorbis-devel libogg-devel libtheora-devel freetype-devel harfbuzz-devel libcurl-devel openssl-devel libsodium-devel sqlite-devel
+  dnf -y install cmake git p7zip gettext rubygem-asciidoctor SDL2-devel physfs-devel libpng-devel openal-soft-devel libvorbis-devel libogg-devel libtheora-devel freetype-devel fribidi harfbuzz-devel libcurl-devel openssl-devel libsodium-devel sqlite-devel
```

Comment:
Probably should be `fribidi-devel` instead?

Reply:
That's right. I [missed it](https://packages.fedoraproject.org/pkgs/fribidi/fribidi-devel/).